projects/plugins/jetpack/modules/widgets/milestone/class-milestone-widget.php

```diff
@@ -283,6 +248,18 @@ public function widget( $args, $instance ) {
 		do_action( 'jetpack_stats_extra', 'widget_view', 'milestone' );
 	}
 
+	/**
+	 * Enqueue widget styles
+	 */
+	public function enqueue_scripts() {
+		wp_enqueue_style(
+			'milestone-widget',
```

Comment:
Since you're creating a new stylesheet, you could opt to include it in the concatenated list of frontend stylesheets included in Jetpack, here:
https://github.com/Automattic/jetpack/blob/96351c0f811f5e1cb658aad1fbc0c441a906af04/projects/plugins/jetpack/class.jetpack.php#L55-L60
https://github.com/Automattic/jetpack/blob/e6f3dc5171e4f23b5e835746b25733ab71715aa8/projects/plugins/jetpack/tools/builder/frontend-css.js#L20-L25

In practice, this means that on most sites, the stylesheet you've created would never be enqueued; instead the css would get included as part of a general `jetpack.css` stylesheet.

Reply:
That's good to know! Thanks! I have applied the changes in the commit https://github.com/Automattic/jetpack/pull/21904/commits/79a650d4cc15bb0bd2cb40e1510924dc910b5b9e.